assets/scss/application.scss

```diff
@@ -10,9 +10,20 @@ $govuk-page-width: $moj-page-width;
 @import './components/footer';
 @import './components/header-bar';
 @import './components/whats-new';
+@import './components/print-link';
 @import './pages/help';
 @import './pages/homepage';
 @import './pages/managed-page';
 @import './pages/whats-new';
 @import './pages/whats-new-post';
+@import './pages/establishment-roll';
 @import 'assets/scss/local';
+@import 'assets/scss/print';
+
+.text-align-right {
```

Comment:
We have a `helpers.scss` file which seems like the current general dumping ground for shared things like this.  Worth putting these in there?